### DISCUSSION TITLE
Longitudinal Analysis, Multivariate Analysis

### DISCUSSION THREAD

**Discussion by GTempl** [FrederickHuangLin/ANCOMBC#50] [Q&A] [answered]
Please forgive if this is redundant or too basic of a question. 
With the coefficient from the primary result, is this a multivariate analysis or a series of univariate analyses within each variable? Statistics is not my strong suit. 

Related question: I have 10 subjects with 2-3 timepoints (samples) each (and a number of covariates of interest). From another discussion, it looks like this analysis is not suited to longitudinal design / mixed effects modeling. Or am I misunderstanding?
Thank you for your time! 
Ginger

**Comment by FrederickHuangLin** [comment, marked as answer]
Hi @GTempl,

Please find below my responses to your questions:

1. Yes, it is a multivariate analysis

2. You are right, ANCOMBC currently is not able to handle longitudinal analysis, we are trying to extend the function by early next year.

Best,
Huang
